msmbuilder/featurizer/multichain.py

```diff
@@ -0,0 +1,510 @@
+# Author: Brooke Husic <brookehusic@gmail.com>
+# Contributors: 
+# Copyright (c) 2016, Stanford University and the Authors
+# All rights reserved.
+
+from __future__ import print_function, division, absolute_import
+
+import warnings
+
+import mdtraj as md
+from mdtraj.utils import ensure_type
+from mdtraj.utils.six import string_types
+from mdtraj.utils.six.moves import xrange
+import numpy as np
+import itertools
+
+from msmbuilder import libdistance
+from . import Featurizer, zippy_maker, dict_maker, featurize_all
+
+
+class LigandFeaturizer(Featurizer):
+    """Base class for objects that featurize protein-ligand trajectories.
```

Comment:
I modeled it off [`Featurizer`](https://github.com/msmbuilder/msmbuilder/blob/master/msmbuilder/featurizer/featurizer.py#L103-L104) and wanted it to be consistent. `Featurizer`shows up in a lot of places, seems like it would be a hassle to change.